specs/boot_loader_specification.md

```diff
@@ -170,8 +170,9 @@ only: while boot loaders should look in both places, OSes should only add their
 files to `$BOOT`.
 
 **Note:** _In all cases the `/loader/entries/` directory should be located
-directly in the root of the file system. Specifically, the `/loader/entries/`
-directory should **not** be located under the `/EFI/` subdirectory on the ESP._
+directly in the root of the ESP or `$BOOT` file system. Specifically, the
```

Comment:
But `$BOOT` can be the ESP, so why not just `$BOOT`?

Reply:
Because it can also not be the ESP, and it that case we look on both the ESP and $BOOT

Reply:
$BOOT is supposed to indicate where we *put* boot entries. It points to either ESP or XBOOTLDR, depending on the local setup, and it only matters when wondering where to drop new stuff.

If we change the perspective, and are *looking* for boot entries, then $BOOT is not the right word to use, in particular as we actually want to look into both ESP *and* XBOOTLDR always. Hence, let's make this clear here, and list *both* dirs under their real names, not under the singleton pointer to one of them.

Hence, I think this should be rewored to say "root of the ESP or XBOOTLDR partition".